docs/projects/reviews/2024/2024-credo-annual.md

```diff
@@ -0,0 +1,65 @@
+
+It would be good for Credo to attract a more diverse set of maintainers from different organizations.
+
+Current state: https://github.com/openwallet-foundation/credo-ts/pull/2134
+State last year: https://github.com/openwallet-foundation/credo-ts/blob/97d617d6bec10dd2e1f1ba7be94a5ab93bd1c628/MAINTAINERS.md
+
+## Project Adoption
+
+Last year Credo already had great adoption by both governments and private organizations. Over the last year the usage of Credo has expanded, especially related to adoption
+
+Credo is being used by goverments in North America ([British Columbia](https://www2.gov.bc.ca/gov/content/governments/government-id/bc-wallet)), organizations in North America ([Northern Block](https://northernblock.io/blog/aip-2-interoperability), [Entidad](https://www.entidad.io/) (through Farmworker Wallet OS)), organizations in South America ([2060.io](https://2060.io/)), governments in Asia ([Bhutan](https://www.biometricupdate.com/202408/bhutans-self-sovereign-digital-id-switches-to-polygon-blockchain) through [Credebl](https://credebl.id/)), organizations in Asia ([Ayanworks](https://www.ayanworks.com/)), governments in Europe (Government of Ireland), organizations in Europe ([Animo](https://animo.id/), [ID Crypt](https://idcrypt.global)), and more. This list is not exhaustive, but based on public announcements governments and organizations have made about using Credo.
+
+Alghtough we would like to see more large scale deployments of Credo (there are some but not many), we're really happy with the diverse adoption of the framework around the world. We think this is partly due to the support for both DIDComm/Aries as well as OpenID4VC standards.
+
+## Goals
+
+### Performance Against Prior Goals
+
+A big goal of Credo over the last year been has been staying on top of latest advandcements in digital credentials. A lot of progress has been made in adding support for OpenID for Verifiable Credentials, SD-JWT VC, mDOC, DCQL (a new credential query langauge) and X509 certificates. Current work is ongoing to support OpenID Federation and in-person exchange of mDOC credentials.
+
+In addition a lot of refactoring has been done and is being done to generalize the architecture of the framework. From being fully Hyperledger Indy / Hyperledger AnonCreds / Hyperledger Aries focussed, to providing the needed interface to support any protocol, ledger or credential format.
+
+Pull requests to add support for DIDComm v2 have been open for over a year, and still haven't been merged. Part of the refactoring that is currently ongoing are the first steps to also get DIDComm v2 merged. We've not seen a lot of need or adoption of DIDComm v2, and therefore it's not been prioritized by any of the current contributors or maintainers.
+
+We didn't specify specific goals last year, but we're really happy with the progress that has been made over the last year. We didn't meet our goals related to writing documentation, making it easeir to adopt the framework.
```

Comment:
```suggestion
We didn't specify specific goals last year, but we're really happy with the progress that has been made over the last year. We didn't meet our goals related to writing documentation, making it easier to adopt the framework.
```

So true. I hope we can put this on the top of the list for the beginning of 2025.